configs/ejenti/trigger_config.schema

```diff
@@ -74,7 +78,8 @@
     "pulse_username",
     "pulse_password",
     "jobs",
-    "log_filter"
+    "log_filter",
+    "suite_white_list_for_signature"
```

Comment:
Due to we assign the default value by following code
```python
self.suite_white_list_for_signature = config.get(TasksTrigger.KEY_CONFIG_SUITE_WHITE_LIST_FOR_SIGNATURE, [])
```
It seems like we don't have to set `suite_white_list_for_signature` as `required` field.